src/rgw/rgw_crypt.cc

```diff
@@ -35,8 +35,6 @@ using namespace rgw;
 /**
  * Encryption in CTR mode. offset is used as IV for each block.
  */
-#warning "TODO: move this code to auth/Crypto for others to reuse."
```

Comment:
warnings were added to move the code to auth/Cryto.
But Have you raised an PR for movement?
Or any existing PR that has done movement.
I believe class AES_256_CTR needed to be moved .. but not done yet